scripts/build_utils.sh

```diff
@@ -837,6 +837,12 @@ gen_debian_version() {
     echo "${raw}${bptag}"
 }
 
+set_extra_buildargs() {
```

Comment:
```suggestion
maybe_set_jaeger() {
```

Reply:
but a better solution is to follow the example of https://github.com/ceph/ceph-build/blob/c696e3df69544d0753f19db47b4d4cf0a611602f/ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml#L157-L174

Reply:
ahan, thanks for pointing out, this is so useful!!